### PR TITLE
Upgrade imports and functions

### PR DESCRIPTION
**Pull request by aayushagrawal135**
Issue #9

### CODE REVIEW

**Comment by aayushagrawal135** [comment]
Hello @claudiashi57, 

I am trying to reproduce neural net training for IHDP experiments locally and required the following changes in the commit. I would be glad if you could review the same.

Thank you.

**Review by aayushagrawal135 — COMMENTED** [review]
The parameter `test_size` could be a float from `0.0` to `1.0` denoting the fraction of `test_size` and could be an `int` denoting the absolute size. However, function documentation says that `0` is not a permissible value. Please let me know if `test_size` as `0.2` makes sense.

**Comment by claudiashi57** [comment]
we used the validation set to set the parameter but the entire dataset to infer the causal effect. The test_size should be zero indeed. There shouldn't be an error if you use the environment described. Otherwise, feel free to comment that line out!
